tests/Baseline/hilti.types.integer.shift-overflow/output

```diff
@@ -0,0 +1,2 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+uncaught exception hilti::rt::Overflow: <no error>
```

Comment:
This is not ideal as it does not include a location. I tried to make use of `hilti::rt::debug::location`, but sadly `hilti/runtime/include/logging.h` includes this header already.

Reply:
Well, I could just forward-declare it.